Makefile

```diff
@@ -0,0 +1,11 @@
+GOROOTDIR := $(shell go env GOROOT)
+
+
+all: help
+help:
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
```

Comment:
.PHONY: serve

Reply:
sorry i should've been more clear, let's add a `.PHONY: <target>` above each target, since none of them are filenames.